docker/Dockerfile.debian

```diff
@@ -0,0 +1,47 @@
+FROM nginxinc/nginx-unprivileged:1.20.2
```

Comment:
Can we pin the digest like we're doing it for the API server? https://github.com/DependencyTrack/dependency-track/blob/master/src/main/docker/Dockerfile#L1